packages/synapse-react-client/src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -221,8 +218,7 @@ export const SynapseNavDrawer: React.FunctionComponent<
   }
 
   const onProjectSearch = (searchTerm: string) => {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    projectSearchJson.queryTerm = searchTerm.split(/[ ,]+/) as any
```

Comment:
Can drop the cast and `// eslint-disable-next-line` because we added the more appropriate cast above.